tasks/tests.py

```diff
@@ -1,3 +1,8 @@
-# from django.test import TestCase
+from tasks.models import Status, Priority, Task, Comment
 
-# Create your tests here.
+def test_update_status(status_1):
+    assert isinstance(status_1, Task)
```

Comment:
I think it will be better to add one more assert that checks if the status updated as expected

Reply:
Where is status_1 defined? couldn't find it

Reply:
also, you can see that the CI check failed because of it:
https://github.com/redhat-beyond/falcon/pull/73/checks#step:6:20